hummingbot/client/config/config_methods.py

```diff
@@ -0,0 +1,19 @@
+from hummingbot.client.config.config_var import ConfigVar
+from typing import Callable
+
+
+def new_fee_config_var(key):
+    return ConfigVar(key=key,
+                     prompt=None,
+                     required_if=lambda x: x is not None,
+                     type_str="decimal")
+
+
+def paper_trade_disabled():
+    from hummingbot.client.config.global_config_map import global_config_map
```

Comment:
Our common practice is to have import statements on top of the file, please don't change this as it will create confusion for others, exceptions being to avoid circular imports.

Reply:
Yes, the reason I did that was for circular import.